### PR TITLE
TELCODOCS-389 - updating revision numbers

### PR DESCRIPTION
**Pull request by tmulquee**
Jira link: https://issues.redhat.com/browse/TELCODOCS-389

Content is 4.10 specific.

Preview: 
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#install-operator-cli_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#performance-addon-operator-monitoring-upgrade-status_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-provisioning-real-time-and-low-latency-workloads_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#performance-addon-operator-verifying-real-time-kernel-installation_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-dry-run_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-mirroring-images-to-custom-registry_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#instruct-the-tests-to-consume-images-from-a-custom-registry_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-mirroring-to-cluster-internal-registry_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#mirroring-different-set-of-images_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-mirroring-images-to-custom-registry_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#instruct-the-tests-to-consume-images-from-a-custom-registry_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-mirroring-to-cluster-internal-registry_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#mirroring-different-set-of-images_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-image-parameters_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-image-parameters_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-ginko-parameters_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-running-the-tests_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-running-hwlatdetect
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-running-cyclictest
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-running-oslat
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-junit-test-output_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-test-failure-report_cnf-master
https://deploy-preview-39295--osdocs.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.html#cnf-performing-end-to-end-tests-using-single-performance-profile_cnf-master

### CODE REVIEW

File: modules/cnf-configure_for_irq_dynamic_load_balancing.adoc
Comment (MarSik) [inline modules/cnf-configure_for_irq_dynamic_load_balancing.adoc:47]: Please avoid using quay. Use the proper RH image pull spec registry.redhat.io/openshift4/cnf-tests-rhel8:v4.10

File: modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc
Comment (MarSik) [inline modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc:131]: This is not a correct change, you would have to replace the full string here, 4.10 will be based on k8s 1.23 for example.

Comment (aireilly) [reply modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc:131]: @marsik on a 4.10 nightly I see: 

```
$ oc get node -o wide
NAME                                                     STATUS   ROLES    AGE   VERSION           INTERNAL-IP   EXTERNAL-IP   OS-IMAGE                                                        KERNEL-VERSION                 CONTAINER-RUNTIME
openshift-master-0.openshift.pool2.practice.redhat.com   Ready    master   10d   v1.22.1+6859754   10.1.196.21   <none>        Red Hat Enterprise Linux CoreOS 410.84.202201101959-0 (Ootpa)   4.18.0-305.30.1.el8_4.x86_64   cri-o://1.23.0-99.rhaos4.10.gitc3131de.el8
openshift-master-2.openshift.pool2.practice.redhat.com   Ready    master   10d   v1.22.1+6859754   10.1.196.23   <none>        Red Hat Enterprise Linux CoreOS 410.84.202201101959-0 (Ootpa)   4.18.0-305.30.1.el8_4.x86_64   cri-o://1.23.0-99.rhaos4.10.gitc3131de.el8
openshift-worker-0.openshift.pool2.practice.redhat.com   Ready    worker   10d   v1.22.1+6859754   10.1.196.24   <none>        Red Hat Enterprise Linux CoreOS 410.84.202201101959-0 (Ootpa)   4.18.0-305.30.1.el8_4.x86_64   cri-o://1.23.0-99.rhaos4.10.gitc3131de.el8
openshift-worker-1.openshift.pool2.practice.redhat.com   Ready    master   10d   v1.22.1+6859754   10.1.196.25   <none>        Red Hat Enterprise Linux CoreOS 410.84.202201101959-0 (Ootpa)   4.18.0-305.30.1.el8_4.x86_64   cri-o://1.23.0-99.rhaos4.10.gitc3131de.el8
```
Will `cri-o://1.23.0-99.rhaos4.10.gitc3131de.el8` be accurate for GA?

Comment (aireilly) [inline modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc:131]:
```suggestion
4.18.0-305.30.1.rt7.102.el8_4.x86_64   cri-o://1.23.0-99.rhaos4.10.gitc3131de.el8
```

Comment (aireilly) [reply modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc:131]: Also update [this line](https://github.com/openshift/openshift-docs/pull/39295/files/28f830dbc189ae28405bff6432a55f7a63a563f7#diff-cb6107e6ffe4c7fb5ef9ebd3ecccc67dc682f0666a4bc337a85b9f42353b2047R122) as follows: 

```
Note the worker with the role `worker-rt` that contains the string `4.18.0-305.30.1.rt7.102.el8_4.x86_64`:
```

Comment (aireilly) [reply modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc:131]: @MarSik how does this look?

**Review by aireilly — COMMENTED** [review]
Tested against a 4.10 nightly and added some comments.

**Comment by ahardin-rh** [comment]
Good work! Just some notes from our meeting. As discussed, we'll need to make sure that these changes get applied to the main branch as well. Typically, we would submit the changes against the main branch and then cherry-pick to enterprise-4.10. Typically the only exception to that is when we're working specifically with release notes files. The commits also need to be squashed into one.

**Comment by tmulquee** [comment]
@ahardin-rh Hi - we've changed the base from enterprise-4.10 to main. Could you please merge, thanks!

**Comment by aireilly** [comment]
Still needs sign-off from @MarSik on changes above

**Comment by tmulquee** [comment]
@MarSiv can you please check these changes and lgtm when you are satisfied.

**Comment by tmulquee** [comment]
@MarSik can you please check these changes and lgtm when you are satisfied?

**Review by MarSik — APPROVED** [review]
/lgtm

**Pull request merged**
